pkg/template/servicebroker/provision.go

```diff
@@ -240,19 +286,16 @@ func (b *Broker) Provision(instanceID string, preq *api.ProvisionRequest) *api.R
 		return api.BadRequest(kerrors.NewNotFound(templateapi.Resource("templates"), preq.ServiceID))
 	}
 
-	// TODO: enable SAR for template - at the moment I think this doesn't work
-	// properly because group information isn't populated in u.
-	/*
-		if err := util.Authorize(b.kc.Authorization().SubjectAccessReviews(), u, &authorization.ResourceAttributes{
-			Namespace: template.Namespace,
-			Verb:      "get",
-			Group:     templateapi.GroupName,
-			Resource:  "templates",
-			Name:      template.Name,
-		}); err != nil {
-			return api.Forbidden(err)
-		}
-	*/
+	// with groups in the user.Info vs. the username only form of auth, we can SAR for get access on template resources
+	if err := util.Authorize(b.kc.Authorization().SubjectAccessReviews(), u, &authorization.ResourceAttributes{
```

Comment:
bear in mind that merging this as is, then, will mean that the "oc cluster up" approach will be broken for using the TSB until the SC is passing the full user info in, right?

I'm not necessarily saying that's a blocker for merging it, but it might mean we just want to see where the SC is at so we can minimize the hoop jumping as well as being in a temporarily broken state.